parts/k8s/kubernetescustomscript.sh

```diff
@@ -179,7 +175,7 @@ if $REBOOTREQUIRED; then
   echo 'reboot required, rebooting node in 1 minute'
   /bin/bash -c "shutdown -r 1 &"
   if [[ $OS == $UBUNTU_OS_NAME ]]; then
-      holdWALinuxAgent "unhold"
+      aptmarkWALinuxAgent unhold
```

Comment:
should this be a background process to return control to the user without waiting for it to finish?